### ISSUE TITLE
Sunburst Chart doesn't work anymore

### ISSUE THREAD

**Issue by k-kedzierski** [googledatastudio/experimental-visualizations#284]
The Sunburst Chart, (source code [here](https://github.com/googledatastudio/experimental-visualizations/tree/master/viz/sunburst)) created by Yulan Lin, doesn't work anymore.

[Here](https://datastudio.google.com/reporting/1rqtvy5I1GjeJNkSxSDKvZfTukzto9PtI/page/MM13) is a sample GDS report using this visualization, showing an error:

![](https://i.imgur.com/yCFFgAE.png)

Please fix the issue or remove the visualization from Featured Visualizations.

**Comment by paquinmathieu** [comment]
This makes me concerned about using community visualizations. I don't want to build a report around a community visualization if it can break and never be repaired. This issues was opened almost 3 months ago and no response. Clear message to me to stay away from community visualization and use the one included. Our reports are created and shared to clients for them to use over a period of 1.5year. Can't imagine having to go through all of them just to change the visualization that is no longer supported.

**Comment by nicoalma** [comment]
Indeed. As now Feb 2nd 2023 this issue hasn't been solved. Difficult to trust on Community Visualizations
